### ISSUE TITLE
New maps

### ISSUE THREAD

**Issue by C4spanier** [dasilvacontin/ag-drift#43]
should make a map selector to go to the old map if wanted too

**Comment by dasilvacontin** [comment]
Hey! First time I see this until now, thank you for the patience! I recently made it so that the track changes more or less every day as you now know.

I agree that it would be nice to have more maps, and to be able to select or vote which one you want to play.

Ideally even a map maker

**Comment by C4spanier** [comment]
Yeah would be nice, keep up the good work! Loving everything so far.

**Comment by C4spanier** [comment]
me and a friend have been playing around with map ideas.

**Comment by dasilvacontin** [comment]
Map selector is now available.

You can use the command /gamemode timeattack 
And then the command /track
or directly, /track 1, /track 2, etc
